android/app/src/main/java/com/expensify/chat/customairshipextender/CustomNotificationProvider.java

```diff
@@ -181,7 +182,9 @@ private void applyMessageStyle(@NonNull Context context, NotificationCompat.Buil
             String name = messageData.get("person").getList().get(0).getMap().get("text").getString();
             String avatar = messageData.get("avatar").getString();
             String accountID = Integer.toString(messageData.get("actorAccountID").getInt(-1));
-            String message = messageData.get("message").getList().get(0).getMap().get("text").getString();
+            
+            // Use the formatted alert message from the backend. Otherwise fallback on the message in the Onyx data.
+            String message = alert != null ? alert : messageData.get("message").getList().get(0).getMap().get("text").getString();
```

Comment:
I don't know a ton about notifications, can you explain why we use the backend message over the Onyx data? Are they different?

Reply:
Yup for the same reason as [here](https://github.com/Expensify/App/pull/27299/files#r1335319486). The text in the report action isn't quite formatted the same as it is in `getModifiedExpenseMessage`